src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -611,7 +611,7 @@ public async Task RestoreDatabase()
                             localConfiguration,
                             ServerStore.NodeTag,
                             cancelToken);
-                        databaseName = await ValidateFreeSpace(localConfiguration, restoreBackupTask);
+                        databaseName = localConfiguration.DatabaseName;
```

Comment:
You can take the database name from `RestoreBackupTaskBase.RestoreFromConfiguration`

Reply:
done